zerver/tests/test_digest.py

```diff
@@ -121,7 +121,7 @@ def test_multiple_stream_senders(self,
         with queries_captured() as queries:
             handle_digest_email(othello.id, cutoff)
 
-        self.assertTrue(29 <= len(queries) <= 30)
+        self.assertTrue(24 <= len(queries) <= 25)
```

Comment:
@showell you added this test in 51aa3138331e6596e713553280280b7c9d5478f7; do you remember why this was a range, not just a singular value?

Reply:
I think the number of queries is different when we run this test alone vs all the tests in this class or the whole test-suite.

Reply:
Well, I would have thought `flush_per_request_caches()` (combined with similar code for memcached and databases that we do for all tests) would have handled that issue.  But maybe not worth worrying about.

Reply:
Yes, I wondered about that too, but didn't dig deeper. 

Running only `test_multiple_stream_senders` seems to need one query more than when running all the tests in `zerver.tests.test_digest.TestDigestEmailMessages`.

Reply:
I think it's worth adding a comment explaining why there's a range, at least.

Reply:
I did a quick diff and this seems like the additional query, when running the single test. 

```sql
SELECT "django_content_type"."id", "django_content_type"."app_label", "django_content_type"."model" FROM "django_content_type" WHERE ("django_content_type"."model" = 'userprofile' AND "django_content_type"."app_label" = 'zerver')
```